app/api/users/index.js

```diff
@@ -60,7 +60,7 @@ router.get('/:id', restricted, async (req, res) => {
           .json({ message: 'This account has been deactivated' });
       }
     }
-
+    user["average_rating"] = user.total_stars/user.total_reviews;
```

Comment:
**nit**: No need to use the subscript notation here, can you do `user.average_rating`?